### PR TITLE
Add line-height to improve line spacing between texts

### LINKED ISSUE #65: [OTHER] [STYLE] Text inside our community section looks kinda congested
### What would you like to share?

# The issue
the texts inside the [our community, gaming, our discussions, who is nhcarrigan] look congested or packed tightly. I think It becomes a little hard to read the text. I think there should be some line spacing between the texts. 

# Screenshots
I tried to just play around in the dev tools and add a line-height of 25px to the h3 tag inside the out community section. I think the text in the after image looks easier to read.

### Before
Screenshot from 2022-09-30 14-35-20

### After
Screenshot from 2022-09-30 14-33-39

# Proposed Changes
I would like to add some line-height to the texts inside the following section above so that there is some space between texts.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by muditchoudhary**
# Pull Request

## Description
Text inside some section like our community, gaming etc. are too congested. I've added some line height to add some  space between the texts.

## Related Issue

Closes #65

### CODE REVIEW

**Comment by muditchoudhary** [comment]
Hey, @naomi-lgbt could please review this PR?
Thank you!!

File: public/index.css [hunk 1]
```diff
@@ -41,6 +41,9 @@ ul {
   padding: 0;
   overflow: hidden;
 }
+#opening h3 {
+    line-height: 25px;
```
Comment (naomi-lgbt) [inline public/index.css:45]:
```suggestion
    line-height: 1.5;
```

File: public/index.css [hunk 2]
```diff
@@ -95,6 +98,9 @@ li {
   font-family: monospace;
   font-size: 1rem;
 }
+.para {
+    line-height: 25px;
```
Comment (naomi-lgbt) [inline public/index.css:102]:
```suggestion
    line-height: 1.5;
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Exact units aren't ideal for responsiveness - let's pass a raw number to `line-height` and let it calculate based on the font size.

**Comment by muditchoudhary** [comment]
@naomi-lgbt fixed!!

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**